schemas/metadata/metaschema/metaschema.1.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Metaschema for validating the structure of the per-doctype schemas in this repository",
+  "id": "http://jsonschema.net",
+  "properties": {
+    "mozPipelineMetadata": {
+      "additionalProperties": false,
+      "description": "Container for per-doctype metadata that can affect how pings are processed in the pipeline",
+      "properties": {
+        "bq_dataset_family": {
+          "description": "NOT YET IMPLEMENTED: The base name for the destination BigQuery dataset; if this is set to 'telemetry', the pipeline will write into 'telemetry_live'",
```

Comment:
All these fields will be informational only to begin with. The idea is that as we actually change the pipeline and ops logic to rely on these fields, we will remove these `NOT YET IMPLEMENTED` warnings.